site/content/deployment/openshift.md

```diff
@@ -10,10 +10,35 @@ to communicate with each other.
 
 ## Prerequisites
 
-The main prerequisite for this kind of deployment is to have an available OpenShift cluster. For a local development, it's pretty simple having such cluster just
-using the OpenShift client tools that can be downloaded from the [OpenShift Origin](https://github.com/openshift/origin/releases) project repository.
+The main prerequisite for this kind of deployment is to have an available OpenShift cluster. For a local development, it's pretty simple having such cluster choosing
+between using :
+
```

Comment:
@sophokles73 I wasn't wrong. Minishift caches the oc tool in a directory that is something like this : /home/ppatiern/.minishift/cache/oc/v1.5.1. If you run the minishift oc-env command it says what is the PATH you have to set in order to use such oc tool binary. 

[ppatiern@new-host ~]$ minishift oc-env
export PATH="/home/ppatiern/.minishift/cache/oc/v1.5.1:$PATH"
#Run this command to configure your shell:
#eval $(minishift oc-env)

if you don't execute that, you are using your own oc tool binary that in my case is in a completely different folder of course.
I think that we should not explain this, because it's part of the Minishift installation that people should do following the official guide. Wdyt ?